tests/src/system/basic/Swift3WhiskObjectTests.scala

```diff
@@ -60,7 +60,7 @@ class Swift3WhiskObjectTests
                     activation.fields("response").asJsObject.fields("success") should be(true.toJson)
 
                     // should have a field named "activationId" which is the date action's activationId
-                    activation.fields("response").asJsObject.fields("result").asJsObject.fields("activationId").toString.length should be >= 32
+                    activation.fields("activationId").toString.length should be >= 32
```

Comment:
i believe this is changing the semantics of the test.
the test is invoking another action - the test is checking that the _result_ of invoking action A contains an activation id. Not that the activation record of A contains an activation id.